nano/node/rpc.cpp

```diff
@@ -3486,8 +3486,36 @@ void nano::rpc_handler::wallet_representative_set ()
 		nano::account representative;
 		if (!representative.decode_account (representative_text))
 		{
-			auto transaction (node.store.tx_begin_write ());
-			wallet->store.representative_set (transaction, representative);
+			{
+				auto transaction (node.store.tx_begin_write ());
+				wallet->store.representative_set (transaction, representative);
+			}
+			// Change representative for all wallet accounts
+			if (request.get<bool> ("update_existing_accounts", false))
+			{
+				std::deque<nano::account> accounts;
```

Comment:
I'm not sure std::deque is the right data structure to use here, it is generally only used when popping from the front is required, I think std::vector should be used.